pkg/metrics/sketch_series.go

```diff
@@ -31,12 +34,44 @@ type SketchSeriesList []SketchSeries
 
 // MarshalJSON serializes sketch series to JSON.
 func (sl SketchSeriesList) MarshalJSON() ([]byte, error) {
+	// We use this function to customize generated JSON
+	// Quite slow, but hopefully this method is called only in the `agent check` command
```

Comment:
I'd put this in the method documentation (on top of `func (sl SketchSeriesList) MarshalJSON() ([]byte, error) {`)

Reply:
Well it's really the function below that's slow, the MarshalJSON itself, when `--full-sketches` is not used, is fine.

Reply:
I think it's still worth clarifying the 2 behaviours in the `MarshalJSON` method documentation - wdyt?